cmd/siac/walletcmd.go

```diff
@@ -505,6 +509,11 @@ func wallettransactionscmd() {
 		outgoingSiacoinsFloat, _ := new(big.Rat).SetFrac(outgoingSiacoins.Big(), types.SiacoinPrecision.Big()).Float64()
 
 		// Print the results.
+		if uint64(txn.ConfirmationTimestamp) != unconfirmedTransactionTimestamp {
+			fmt.Printf(time.Unix(int64(txn.ConfirmationTimestamp), 0).Format("2006-01-02 15:04:05-0700"))
```

Comment:
`time.Stamp` may be a more human-friendly format. It doesn't include the year, but that might be ok

Reply:
My use case is figuring out my transactions for tax purposes. Maybe not the common case, but also I don't think a terribly unusual use case.

For reconciling financial transactions, the year and time zone are pretty important. `time.Stamp`'s time zone is only an implicit one, but I'd like to avoid the ambiguity.